.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - 6
   - 8
   - 9
```

Comment:
Can we also remove version 9 from the CI since it also reached end-of-life almost a year ago now?